test/contracts/common/unstructured_storage.js

```diff
@@ -23,12 +23,12 @@ contract('Unstructured storage', () => {
       // checks
       assert.equal(
         await web3.eth.getStorageAt(appStorage.address, (await appStorage.getKernelPosition())),
-        (await appStorage.kernel()).toString(),
+        (await appStorage.kernel()).toString().toLowerCase(),
         'Kernel should match'
       )
       assert.equal(
         await web3.eth.getStorageAt(appStorage.address, (await appStorage.getKernelPosition())),
-        kernel.address,
+        kernel.address.toLowerCase(),
```

Comment:
I'm starting to wonder if we should have some sort of `assertAddressEquals()` or etc. so we don't have to do this :smile:

Reply:
Haha this one was a bit particular